.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
If you want to add this file, please do this in a separate commit.

Reply:
oops, wasn't supposed to happen

Reply:
Removed